src/modules/addtask.js

```diff
@@ -0,0 +1,14 @@
+function saveTasks(tasktodo) {
+  localStorage.setItem('tasktodo', JSON.stringify(tasktodo));
+}
+
```

Comment:
well done , I would just suggest that you put this function in a separate module that contains localstorage manipulation and import it , or just call  localStorage.setItem('tasktodo', JSON.stringify(tasktodo)); directly since it is one statement